src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -62,6 +62,23 @@ object JmapMessages {
     jsonPath(messageIdsPath).findAll.saveAs("messageIds")
   }
 
+  def moveSentMessagesToMailboxId = 
+    foreach("${messageIds}", "messageId") {
```

Comment:
why do you want to issue a query for each message ?

Reply:
That's not really easy, _foreach_ is a gatling method & I don't succeed to chain to something else than gatling methods.